index.js

```diff
@@ -4,7 +4,36 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+// error handling test
+let result = 0;
+if(process.argv.length < 3) {
+ return 'No operation provided...'
+}
+else if(!process.argv[3]) {
+ return 'No numbers provided...'
+}
 
+// plus test
+if(process.argv[2] === 'plus') {
+    for(let i = 3; i < process.argv.length; i++) {
+        result += Number(process.argv[i])
+    }
+    return result
+}
+// minus test
+if(process.argv[2] === 'minus') {
+result = process.argv[3]
+    for(let i = 4; i < process.argv.length; i++) {
+        result -= Number(process.argv[i]);
+}
+    return result
+}
```

Comment:
style comment: try to follow indentation rules as much as you can! It'll make your code easier to read

Reply:
I always forget! thank you Myra!